src/server/dfly_bench.cc

```diff
@@ -452,27 +486,58 @@ void TLocalClient::Connect(tcp::endpoint ep) {
     fb.Join();
 }
 
-void TLocalClient::Run(uint32_t key_min, uint32_t key_max, uint64_t cycle_ns) {
-  KeyGenerator key_gen(key_min, key_max);
-  CommandGenerator cmd_gen(&key_gen);
+void TLocalClient::Start(uint32_t key_min, uint32_t key_max, uint64_t cycle_ns) {
```

Comment:
this is an async function that only creates all the running fibers but does not run them

Reply:
It's not really async on its own, right? It just creates other async Fibers

Reply:
right